src/dataflow-types/src/logging.rs

```diff
@@ -52,6 +52,7 @@ pub enum MaterializedLog {
     DataflowCurrent,
     DataflowDependency,
     FrontierCurrent,
+    KafkaBrokerRTT,
```

Comment:
optional nit: Rust style for acronyms would be `Rtt`

Reply:
Thanks for pointing this out! I had figured that because RTT is an initialism, it would be `RTT` and not `Rtt`. However, it seems that the style guide doesn't distinguish between the two:

> In CamelCase, acronyms count as one word: use Uuid rather than UUID.

Frank's pet peeve strikes again!

I'm going to go ahead and make the change 😄